google/cloud/sql/connector/instance.py

```diff
@@ -134,6 +134,13 @@ def get_preferred_ip(self, ip_type: IPTypes) -> str:
         )
 
 
+def _format_user_agent(version: str, driver: str, custom: Optional[str]) -> str:
+    agent = f"{APPLICATION_NAME}/{version}+{driver}"
+    if custom:
+        agent = " ".join([agent, custom])
```

Comment:
is it normal to have a whitespace in a user agent? Would this make this harder to parse for internal tools?

Reply:
Yes. It's a standard. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent. We do the same in the Proxy. https://github.com/GoogleCloudPlatform/cloud-sql-proxy/blob/main/cmd/root.go#L656